src/cljc/witan/schema.cljc

```diff
@@ -19,3 +20,101 @@
   "validation for /login"
   {(s/required-key :username) (s/both (length-greater 5) (is-an-email))
    (s/required-key :password) (length-greater 5)})
+
+(def IdType
+  s/Str)
```

Comment:
are we going with uuid type or str?
still need to find out the advantages of one vs the other